MAINTAINERS.yml

```diff
@@ -2115,6 +2115,22 @@ nRF Platforms:
   labels:
     - "platform: nRF"
 
+Renesas Platforms:
+  status: maintained
+  maintainers:
+    - andrzej-kaczmarek
+  files:
+    - boards/arm/da14*/
+    - drivers/*/*smartbond*
+    - dts/arm/renesas/smartbond/
+    - dts/bindings/*/renesas,smartbond*
+    - soc/arm/renesas_smartbond/
+  labels:
+    - "platform: Renesas"
+  description: >-
+    Renesas SOCs, dts files, drivers and related boards not belonging to the
```

Comment:
Renesas Smartbond SOCs, dts files and related drivers. Renesas boards based on Smartbond SOCs.